Dangerfile

```diff
@@ -1,3 +1,12 @@
+# Make it more obvious that a PR is a work in progress and shouldn't be merged yet.
+has_wip_label = github.pr_labels.any? { |label| label.include? "WIP" }
+has_wip_title = github.pr_title.include? "[WIP]" || github.pr_title.include? "WIP"
```

Comment:
can we check 1 more thing @byhbt :D checking is_draft PR 😁  so we can cover all the cases :D

Reply:
I think this case can cover the previous one
`github.pr_title.include? "WIP"`

Reply:
And this maybe helps you to find the draft state
https://spectrum.chat/danger/javascript/support-for-github-draft-prs~82948576-ce84-40e7-a043-7675e5bf5690